index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="header-nav">
+        <a href="/">
+          <div class="header-logo">
+            <img src="images/logo.svg" /><img src="images/logo-txt.svg" /></div
+        ></a>
+        <a href="/login" class="header-login">로그인</a>
+      </div>
+    </header>

```

Comment:
```suggestion
    <header>
        <a href="/">
          <div class="header-logo">
            <img src="images/logo.svg" /><img src="images/logo-txt.svg" /></div
        ></a>
        <a href="/login" class="header-login">로그인</a>
    </header>
```
- 현재 css 스타일을 봤을때에는 `div`의 역할을 `header`가 할 수 있는 것 처럼 보입니다. 의미없이 태그를 한단계 더 들여 쓸 필요는 없습니다.